### PR TITLE
feat(service-portal-applications): Draft status bar for application list 

### PR DESCRIPTION
**Pull request by egillben**
# Draft status bar

https://app.clickup.com/t/2z5fv2n

## What
Draft status bar for applications with steps finished / total steps

## Why
So users can see their progress in the draft state in a better manner.
(old way was obsolete and not functional)

## Screenshots / Gifs
image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by disaerna — APPROVED** [review]
Code owned file looks good

File: libs/application/templates/public-debt-payment-plan/src/forms/PrerequisitesForm.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const PrerequisitesForm: Form = buildForm({
   id: 'PrerequisitesForm',
   title: application.name,
   logo: Logo,
-  mode: FormModes.DRAFT,
+  mode: FormModes.NOT_STARTED,
```
Comment (obmagnusson) [inline libs/application/templates/public-debt-payment-plan/src/forms/PrerequisitesForm.ts:37]: Might want to mention this setup in the readme.

**Review by obmagnusson — CHANGES_REQUESTED** [review]
Minor comment

File: libs/application/core/README.md [hunk 2]
```diff
@@ -168,6 +168,7 @@ You can see a simple example from the reference-template [here](https://github.c
 
 We have a few different statuses that an application can be in. They are:
 
+- `notstarted` The user has opened the application / it is saved in the database, but is not listed yet (etc still in prerequisites)
```
Comment (obmagnusson) [inline libs/application/core/README.md:171]:
```suggestion
- `notstarted` The user has opened the application / it is saved in the database, but is not listed yet (eg. still in prerequisites)
```

**Review by obmagnusson — APPROVED** [review]
LGTM

**Review by albinagu — APPROVED** [review]
island-ui lgtm

**Pull request merged**